recipe/meta.yaml

```diff
@@ -32,13 +32,13 @@ requirements:
     run:
         - python
         - numpy x.x
-        - libpng
+        - libpng 1.6*
         - jasper
-        - jpeg
-        - zlib
-        - libnetcdf
+        - jpeg 9*
+        - zlib 1.2*
+        - libnetcdf 4.3*
         - hdf4
```

Comment:
`hdf4` version available in both `defaults` and `conda-forge` is `4.2.11`.
`jasper` must be pinned to `jpeg 9*` and `jpeg` can be removed from here. Jasper is not updated since 2007! So I think that is pretty much what we have and will have for life :wink: